src/assets/index.html

```diff
@@ -16,9 +16,15 @@
         font-weight: normal;
       }
 
-      body, ::placeholder {
+      body {
         font-size: 85%;
       }
+
+      ::placeholder {
+        font-size: 85%;
+        color: #999 !important;
```

Comment:
`!important` is maybe the bad. but... the placeholder text was looking like it was actual text. 🤷

Reply:
I trust your css hackery.

Reply:
haha. dangerous business if I'm writing css. 😬